core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -19,9 +19,17 @@ public interface Configs {
   String DEFAULT_ENTITY_KV_STORE = "RocksDBKvBackend";
   String ENTITY_KV_STORE_KEY = "gravitino.entity.store.kv";
 
-  String DEFAULT_ENTITY_RELATIONAL_STORE = "MySQLBackend";
+  String DEFAULT_ENTITY_RELATIONAL_STORE = "JDBCBackend";
   String ENTITY_RELATIONAL_STORE_KEY = "gravitino.entity.store.relational";
 
+  String ENTITY_RELATIONAL_JDBC_BACKEND_TYPE_KEY = "gravitino.entity.store.relational.jdbcType";
+  String DEFAULT_ENTITY_RELATIONAL_JDBC_BACKEND_TYPE = "mysql";
```

Comment:
`mysql`-> `jdbc`?

Reply:
Here is a database type parameter that supports JDBC connection. Users can specify the supported database type. We now support MySQL, so this is in line with expectations.

Reply:
Have changed parameters to make them more readable